.github/ISSUE_TEMPLATE/release.md

```diff
@@ -23,20 +23,28 @@ examples of each step, assuming release vX.Y.0 is being cut.
 
 - [ ] **[In Crossplane Runtime]**: Prepared the release branch `release-X.Y` at the beginning of [Code Freeze]:
```

Comment:
this list is getting long enough now that we may want to separate it into sections, e.g.:

## Code Freeze

## Release 

or whatever sections are best applicable to the flow of this list. Those two come to mind immediately though, esp. since they are done at different times.